### DISCUSSION TITLE
6.1.11 SDRTrunk Alias is stronger than Rdio-Scanner?

### DISCUSSION THREAD

**Discussion by noamlivne** [chuot/rdio-scanner#130] [General]
I am testing now 6.1.11 with Windows 10 and SDRTrunk.
Unfortunately, even though I change the Label/Name in Rdio-Scanner and Save it, it reverts to the Alias of SDRTrunk. 
I do not think that was the case till now.
I always had the Auto-populate On, and always Rdio-Scanner's Label/Name stayed the same if I saved them, and were not replaced by SDRTrunk Aliases.
Is this now by-design, or can Rdio-Scanner be "stronger" and keep the Name/Label I save there, regardless of SDRTrunk?
Thank you,
And sorry for the trouble.

**Comment by chuot** [comment]
You should upgrade to v6.1.12 which fix a major issue.

It is by design if you set the auto-populate option (which is by default). If you want to lock the tg labels, then you must turn off the auto-populate option and create the tgs manually.
